Formula/go.rb

```diff
@@ -20,6 +20,15 @@ class Go < Formula
     sha256 "4ae792ffe880c4658a596a82fa8276d39d78d2e7b64e7ac9598649f0eefed90c" => :el_capitan
   end
 
+  devel do
+    url "https://dl.google.com/go/go1.10beta1.src.tar.gz"
+    sha256 "841df62b20fd915d83a2e43b7d043c2a3781c299de78abc45480eec575186b6b"
+
+    resource "gotools" do
+      url "https://go.googlesource.com/tools.git"
```

Comment:
I think we need to pin the branch here

Reply:
This repo doesn't currently have a release branch for 1.10. Should I pin it to `master`?

In looking through the history of this file, when a devel release is introduced, the branch hasn't been pinned (probably due to the aforementioned issue).

Reply:
Ah. I didn't realize there's no 1.10 branch even though there's a 1.10 beta :see_no_evil: